### DISCUSSION TITLE
Unable To Connect TO DB With TablePlus

### DISCUSSION THREAD

**Discussion by Kailashlambe** [wardenenv/warden#593] [Help & Support]
Hi Team,

I successfully set up the Magneto 2 project, when I tried to connect DB by using Table Plus, I get the error like `can not connect to the host` .

I have followed the documentation https://docs.warden.dev/configuration/database.html, and I have tried the SSH host tunnel details with generic connection details like,

![image](https://user-images.githubusercontent.com/10921176/218026364-b30a609e-4344-4530-9d40-ecf3c00a5823.png)

and table plus connection details like, 

![image](https://user-images.githubusercontent.com/10921176/218026281-512e8813-0f2e-4e5b-a904-9fb469d90d2b.png)

But no luck.

Please guide.

**Comment by dudzio12** [comment]
You need to provide ssh_key, usually located in `~/.den/tunnel/ssh_key`

**Reply by Kailashlambe** [reply]
Hi @dudzio12,

Thanks for the feedback.

I have already tried with ssh_key too but no luck.
